Prolific/Prolific/Views/Cells/Round Cell/RoundCell.m

```diff
@@ -10,10 +10,42 @@
 
 @implementation RoundCell
 
+- (instancetype)initWithFrame:(CGRect)frame
+{
+    self = [super initWithFrame:frame];
+    if (self) {
+        _cellView = [[RoundCellView alloc] initWithFrame:frame];
+        [self.contentView addSubview:_cellView];
+    }
+    return self;
+}
+
+#pragma mark - Setup
+
 - (void)layoutSubviews {
     [super layoutSubviews];
     
-    // TODO: add other labels to Round cell
+    _cellView.frame = self.contentView.bounds;
+    
+    if (_snippet) {
```

Comment:
one piece of feedback here. Maybe the view itself should handle the hiding and unhiding.

For example the snippet could have a state variable that is called `showingSnippet` or something. Then here are you would do is say `_cellView.ShowingSnippet = NO / YES;`

Right now you are reaching in and editing grandchildren (your child view's child views).